airflow/providers/vertica/operators/vertica.py

```diff
@@ -48,5 +48,5 @@ def __init__(
 
     def execute(self, context: 'Context') -> None:
         self.log.info('Executing: %s', self.sql)
-        hook = VerticaHook(vertica_conn_id=self.vertica_conn_id)
+        hook = VerticaHook(vertica_conn_id=self.vertica_conn_id, log_sql=False)
```

Comment:
Wouldn't removing `self.log.info('Executing: %s', self.sql)` line be easier?

Reply:
I tihnk keep it in operator is better - it will be logged in our logger and will get to the "airflow.task" output.

Reply:
Ah but it is hook, so yeah maybe better.

Reply:
But it really doesn't matter :)